pkg/apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -344,8 +344,9 @@ type SaveAsTemplateOutput struct {
 }
 
 type EKSStatus struct {
-	UpstreamSpec   *eksv1.EKSClusterConfigSpec `json:"upstreamSpec"`
-	VirtualNetwork string                      `json:"virtualNetwork"`
-	Subnets        []string                    `json:"subnets"`
-	SecurityGroups []string                    `json:"securityGroups"`
+	UpstreamSpec     *eksv1.EKSClusterConfigSpec `json:"upstreamSpec"`
+	VirtualNetwork   string                      `json:"virtualNetwork"`
+	Subnets          []string                    `json:"subnets"`
+	SecurityGroups   []string                    `json:"securityGroups"`
+	LaunchTemplateID string                      `json:"launchTemplateId"`
```

Comment:
there could be multiple launch template ID's right? or is this just the one that we create?

Reply:
This is just the one we create and keep track of.